activerecord-pluck_in_batches.gemspec

```diff
@@ -20,5 +20,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "activerecord", ">= 4.2.7", "< 6.0.0"
 
   spec.add_development_dependency "bundler", "~> 2.0"
-  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "rake", ">= 10.0", "< 13.0.0"
```

Comment:
Locked to `13.0.0` for the same reasons as above. It might not be necessary, but anything higher is untested. It's easy to change when we need open it up for versions greater than 13.